### PR TITLE
Display institution name on comments

### PR DESCRIPTION
**Pull request by ruanres**
issue #146

### CODE REVIEW

File: app/comment/commentService.js [hunk 1]
```diff
@@ -18,9 +18,10 @@
             return deferred.promise;
         };
 
-        service.createComment = function createComment(postKey, text) {
+        service.createComment = function createComment(postKey, text, institutionKey) {
             var deferred = $q.defer();
-            var data = {'text': text};
+            var data = {'text': text, 'institution_key': institutionKey};
+            console.log(data);
```
Comment (mayzabeel) [inline app/comment/commentService.js]: remover

File: app/post/post_details.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
               <div>
                 <a href class="md-title hyperlink" ng-click="postDetailsCtrl.goToInstitution(post.institution_key)">{{ post.institution_name }}</a>
               </div>
-              <span class="md-subhead">por {{ post.author }} em {{ post.publication_date | date:'dd-MM-yyyy'}} às {{post.publication_date | date:'HH:mm:ss'}}</span>
+              <span class="md-subhead">por {{ post.author }} em {{ post.publication_date | date: 'MMM d, y'}} às {{post.publication_date | date:'HH:mm:ss'}}</span>
```
Comment (mayzabeel) [inline app/post/post_details.html]: acho que tu mecheu sem querer aqui

File: models/post.py [hunk 3]
```diff
@@ -37,6 +37,12 @@ class Comment(ndb.Model):
     # user who is the author
     author = ndb.KeyProperty(kind="User", required=True)
 
+    # institution which the author is representing
+    institution_key = ndb.KeyProperty(kind="Institution", required=True)
```
Comment (mayzabeel) [inline models/post.py]: remover

File: models/post.py [hunk 4]
```diff
@@ -63,9 +74,12 @@ def make(comment):
         """Create personalized json of comment."""
         publication_date = comment.publication_date.isoformat()
         author = comment.author.get()
+        # TODO: rever isto acima
```
Comment (mayzabeel) [inline models/post.py]: rever o que?

File: models/post.py [hunk 5]
```diff
@@ -48,12 +51,16 @@ def create(data, author_key, post_key):
         """Create a comment and check required fields."""
         if not data['text']:
             raise FieldException("Text can not be empty")
+        if not data['institution_key']:
+            raise FieldException("Instituion can not be empty")
```
Comment (andreldsa) [inline models/post.py]: Institution*

**Pull request merged**